frontend/src/components/landing/DemoPhone.tsx

```diff
@@ -34,12 +36,19 @@ export const DemoPhone = (props: Props) => {
       screenshot = PremiumScreenshotDe.src;
     }
   }
+  let foreground = FgImage.src;
+  if (lang === "fr") {
+    foreground = FgImageFr.src;
+  }
+  if (lang === "de") {
+    foreground = FgImageDe.src;
+  }
```

Comment:
Suggestion (non-blocking) kill `let` with fire ...
```suggestion
  const foreground = lang === "de" ? FgImageDe.src : lang === "fr" ? FgImageFr.src : FgImage.src;
```

Though I think maybe in the past I asked to REMOVE a nested ternary expression? I guess I dislike `let` more than I dislike nested ternary expressions.